src/components/Loader/Loader.jsx

```diff
@@ -1,23 +1,34 @@
 import React, { forwardRef } from "react";
 import PropTypes from "prop-types";
 import "./Loader.scss";
+import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
 
-const Loader = forwardRef(({ svgClassName }, ref) => {
-  return (
-    <div className="monday-loader-component" ref={ref} role="alert" title="loading">
-      <svg className={`circle-loader-spinner ${svgClassName}`} viewBox="0 0 50 50">
-        <circle className="circle-loader-spinner-path" cx="25" cy="25" r="20" fill="none" strokeWidth="5" />
-      </svg>
-    </div>
-  );
-});
+const Loader = forwardRef(
+  (
+    {
+      // Backward compatibility for props naming
+      svgClassName,
+      className
+    },
+    ref
+  ) => {
+    const overrideClassName = backwardCompatibilityForProperties([className, svgClassName], "");
```

Comment:
undefined create here bad class name in {`circle-loader-spinner ${overrideClassName}` when empty